packages/hash/frontend/src/components/layout/MainContentWrapper.tsx

```diff
@@ -30,7 +45,7 @@ const Main = styled("main", {
   }),
 }));
 
-export const MainContentWrapper: FunctionComponent = ({ children }) => {
+export const MainContentWrapper: FC = ({ children }) => {
```

Comment:
I don’t think we’ll be able to use `FC` in React 18 typings – they remove implicit children there. Just something to keep in mind, no need to change anything in this PR.

Reply:
Noted.

Reply:
There are still a few cases of FC instead of VFC, but we can bulk-replace them later.